mainapp/api/views.py

```diff
@@ -2,19 +2,24 @@
 from ..models import Bot
 from .serializers import BotSerializer
 from rest_framework.decorators import api_view
+import logging
+
+logger = logging.getLogger('root')
```

Comment:
best practice is do 
```suggestion
logger = logging.getLogger(__name__)
```
then you can tweak this logger separately from root one